biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java

```diff
@@ -1830,6 +1830,11 @@ public OnWorkspace on(String ownerName) {
 	 * @return true if the properties were changed
 	 */
 	public boolean doExtend(Processor processor) {
+		PluginsContainer plugins = processor.getPlugins();
+		if (!plugins.getInterfaces()
+			.isEmpty()) {
+			getExternalPlugins().setAbstractPlugins(processor, plugins);
+		}
```

Comment:
In Workspace.propertiesChanged, it does:

https://github.com/bndtools/bnd/blob/879e8cdc025b59d1e62664805f02293cce824199/biz.aQute.bndlib/src/aQute/bnd/build/Workspace.java#L508-L511

Processor.propertiesChanged clears the plugsin:

https://github.com/bndtools/bnd/blob/879e8cdc025b59d1e62664805f02293cce824199/biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java#L1021-L1027

So doesn't that discard the "abstract" plugins?